### PR TITLE
documentation: Add SQL Request on index_data in known inssue

### PR DESCRIPTION
**Pull request by jeremyjaouen**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [x] 21.04.x (master)

### CODE REVIEW

File: en/graph-views/known-issues.md [hunk 1]
```diff
@@ -22,3 +22,5 @@ issues for future releases.
...
 | Polygon resize may not work as expected                                                                                                                                                                                    | We advice you to recreate them if the rendering fails on the web client                                                                                                                                                                                                                                                 |
+| You have reach the maximum of id into centreon_storage  index_data | Play the following request into MySQL / MariaDB :<br /> ```ALTER TABLE index_data MODIFY id bigint unsigned AUTO_INCREMENT;```<br /> ```ALTER TABLE metrics MODIFY index_id bigint unsigned;```<br /> Depending on your metrics volumes, this operation could be more or less longer.
```
Comment (kduret) [inline en/graph-views/known-issues.md:25]:
```suggestion
| You have reached the maximum of id into centreon_storage.index_data
| Play the following request into MySQL / MariaDB :
| `ALTER TABLE index_data MODIFY id bigint unsigned AUTO_INCREMENT;`
| `ALTER TABLE metrics MODIFY index_id bigint unsigned;`
| Depending on your metrics volumes, this operation could be more or less longer.
```

File: en/graph-views/known-issues.md [hunk 2]
```diff
@@ -22,3 +22,5 @@ issues for future releases.
...
+|
```
Comment (kduret) [inline en/graph-views/known-issues.md:26]:
```suggestion
```

**Comment by lpinsivy** [comment]
@jeremyjaouen donn't forget fr/graph-views/known-issues.md

**Review by lpinsivy — APPROVED** [review]

File: fr/graph-views/known-issues.md [hunk 3]
```diff
@@ -21,4 +21,5 @@ issues for future releases.
...
+| Polygon resize may not work as expected                                                                                                                                                                                    | We advice you to recreate them if the rendering fails on the web client                                                                                                                                                                                                                                  | Vous avez atteinds le maximum d'id dans la table centreon_storage.index_data | Jouer la requête suivante dans MySQL / MariaDB :<br /> `ALTER TABLE index_data MODIFY id bigint unsigned AUTO_INCREMENT;`<br /> `ALTER TABLE metrics MODIFY index_id bigint unsigned;`<br /> Selon la volumétrie de vos métriques, cette opération peut être plus ou moins longue.
```
Comment (kduret) [inline fr/graph-views/known-issues.md:24]:
```suggestion
| Polygon resize may not work as expected                                                                                                                                                                                    | We advice you to recreate them if the rendering fails on the web client
|
| Vous avez atteint le maximum d'id dans la table centreon_storage.index_data | Exécutez la requête suivante dans MySQL / MariaDB :<br /> `ALTER TABLE index_data MODIFY id bigint unsigned AUTO_INCREMENT;`<br /> `ALTER TABLE metrics MODIFY index_id bigint unsigned;`<br /> Selon la volumétrie de vos métriques, cette opération peut être plus ou moins longue.
```

**Review by kduret — COMMENTED** [review]
graph-views section seems more related to centreon map extension

**Review by quanghungb — APPROVED** [review]

**Pull request merged**